libraries/docker/static-files/src/openapi/.gitignore

```diff
@@ -0,0 +1,4 @@
+# Dir for bundles
+dist
+node_modules
+.idea
```

Comment:
I feel like this project looks like something that belongs in eo-base? is this an application, that needs to be able to run, locally or in the cluster?